src/action/roamHelper.ts

```diff
@@ -45,7 +45,7 @@ export function updateCenterAndZoom(
     }
 ) {
     const previousZoom = view.getZoom();
-    const center = view.getCenter();
+    const center = view.getDefaultCenter();
```

Comment:
I personally think the `center` option of the `graph` series is used to set the initial center and both the documentation and the source code don't indicate that it can be a string array. But it seems the [`gauge`](https://github.com/apache/echarts/blob/master/src/chart/gauge/GaugeView.ts#L47-L52) series and [`pie`](https://github.com/apache/echarts/blob/master/src/chart/pie/pieLayout.ts#L44-L57) series support it. If necessary, it might be practical to normalize the `center` at https://github.com/apache/echarts/blob/master/src/chart/graph/createView.ts#L91

Reply:
Could we normalize `center` at `View`
https://github.com/apache/echarts/blob/master/src/coord/View.ts#L130

Looks like `Tree` and `Geo` also have this issue